service/dependencies.gradle

```diff
@@ -23,7 +23,7 @@ dependencies {
 
   // Terra deps
   implementation group: "bio.terra", name: "datarepo-client", version: "2.13.0-SNAPSHOT"
-  implementation group: "bio.terra", name:"billing-profile-manager-client", version: "0.1.536-SNAPSHOT"
+  implementation group: "bio.terra", name:"billing-profile-manager-client", version: "0.1.548-SNAPSHOT"
```

Comment:
I don't think this is actually necessary since we don't actually use BPM's models, but it shouldn't hurt to update